functions-app/calculate-score-v2/v1.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const moment = require('moment')
+
+const config = require('../config')
+const checkWindowDataService = require('./check-window.data.service')
+const schoolScoresDataService = require('./schools-scores.data.service')
+const schoolDataService = require('./school.data.service')
+const pupilResultsDiagnosticCache = require('./pupil-results-diagnostic-cache.data.service')
+const redisCacheService = require('../lib/redis-cache.service')
+
+let logger
+
+const v1 = {
+  process: async function (context) {
+    logger = context.log
+    redisCacheService.setLogger(logger)
+    await handleStoreSchoolsScores(context)
+  }
+}
+
+async function handleStoreSchoolsScores (context) {
+  const liveCheckWindow = await checkWindowDataService.sqlFindCalculationPeriodCheckWindow()
+
+  // Terminate execution if a check window is not within the calculation period
+  if (!liveCheckWindow || !liveCheckWindow.id) {
+    context.log(`calculate-score-v2 v1: Live check window not found`)
+    return
+  }
+
+  const schoolIds = await schoolDataService.sqlFindSchoolIds()
```

Comment:
are we just blindly processing all schools,  or could we be more selective?

Reply:
@GuyHarwood my impression is that we do need to go through all schools